platform/linux-generic/arch/default/odp_wait_until.h

```diff
@@ -0,0 +1,49 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2023 Nokia
+ */
+
+#ifndef ODP_DEFAULT_WAIT_UNTIL_H_
+#define ODP_DEFAULT_WAIT_UNTIL_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <odp/api/cpu.h>
+
+#include <stdint.h>
+
+static inline void _odp_sevl(void)
+{
+	/* empty */
+}
+
+static inline int _odp_wfe(void)
+{
+	return 1;
+}
+
+#define _odp_monitor_u8(addr, mo) __atomic_load_n((addr), (mo))
+#define _odp_monitor_u32(addr, mo) __atomic_load_n((addr), (mo))
+#define _odp_monitor_u64(addr, mo) __atomic_load_n((addr), (mo))
+#define _odp_monitor_u128(addr, mo) __atomic_load_n((addr), (mo))
+
+static inline void _odp_wait_until_not_equal_rel_u32(uint32_t *val, uint32_t expected)
```

Comment:
I think the _rel_ part of the name is misleading since typically in odp _rel_ means release memory ordering (to pair with _acq_). So maybe this should be just _odp_wait_until_not_equal_u32.

Reply:
True, will update this and similar ones in V2.